workflow/setup_expt.py

```diff
@@ -455,6 +463,9 @@ def input_args(*argv):
                       default=os.path.join(_top, 'parm/config/gefs'))
     gefs.add_argument('--yaml', help='Defaults to substitute from', type=str, required=False,
                       default=os.path.join(_top, 'parm/config/gefs/yaml/defaults.yaml'))
+    gefs.add_argument('--icsdir', help='full path to initial condition directory
```

Comment:
```suggestion
    gefs.add_argument('--icsdir', help='full path to initial condition directory [temporary hack in place for testing]',
```
Missed end quote before line break

Reply:
addressed